_includes/events.html

```diff
@@ -7,23 +7,22 @@ <h2 class="section-heading">Events</h2>
     </div>
     <div class="row">
       {% for event in site.events %}
+
       <div class="col-lg-10 col-lg-offset-1 event">
         <div class="col-lg-6">
           <a href={{event.url}}>
             <h1>{{ event.title }}</h1>
           </a>
         </div>
 
-        <div class="col-lg-6">
+        <div class="col-lg-3">
           {% include event-rsvp.html rsvp=event.rsvp %}
         </div>
 
         <div class="col-lg-12">
         <h4>
           Join us {{ event.event-date }}
-          {% if event.location != nil %}
-            at {{ event.location }}
-          {% endif %}
+            at {{ event.location-name }}
```

Comment:
This conditional is here to cover the case where a location-name is not included in the event file yaml frontmatter. 

With this change, if there's no "location-name" it will read "Join us October 6th at".